Docsite template: make it ready-to-use, part 1 (#8)
ansible-community/project-template: README.md

This template gives you the basic recommended (and in some instances required) files for your new Ansible community project.

## Template structure

This template includes sample files for the following:
- [README.md](README.md) - This file. It should describe the project and list the documentation site, when available, and how to reach the project team (Matrix room, if available and Ansible forum tags). 
- [LICENSE.md](LICENSE.md) - The project license. We recommend GPLv3

> - [CONTRIBUTING.md](CONTRIBUTING.md) - The basics for contributing to your project (if there's a docsite, refer to the docsite contributor guide from the file)

Reviewer: ```suggestion
- [CONTRIBUTING.md](CONTRIBUTING.md) - The basics for contributing to your project. If your project has a docsite, refer to the docsite contributor guide from this CONTRIBUTING.md file.
```

To learn how to contribute to this project, see the [Contributor guidelines](https://link-to-docsite-or-contributor.md).

### Getting started development guide

> If the project doesn't have the guide, please add it to mitigate the entry threshold for new contributors. If it's not applicable, remove  the section.

> You have a fix and want to submit a ready-for-merge pull request? See the [Getting started development guide](https://link-to-the-quide).

Reviewer: ```suggestion
Do you have a fix and want to submit a ready-for-merge pull request? See the [Getting started development guide](https://link-to-the-quide).
```

## Communication

> Fill up the following sub-sections with async and real-time channels relevant to your project. Please leave all non-project-specific links untouched.

> > If your project has a docsite, this section should refer to a corresponding docsite section containing the below information as on the [docsite index template page](https://github.com/ansible-community/project-template/blob/main/docs/index.md).

Reviewer: ```suggestion
> If your project has a docsite, this section should refer to a corresponding docsite section that contains the following information as on the [docsite index template page](https://github.com/ansible-community/project-template/blob/main/docs/index.md).
```

## Governance

> > Update this section with relevant information and URLs. If the project has a docsite, have this information there.

Reviewer: ```suggestion
> Update this section with relevant information and URLs. If the project has a docsite, include this information in the docsite.
```
